bundle/manifests/argocd-operator.clusterserviceversion.yaml

```diff
@@ -1901,7 +1901,7 @@ spec:
                       fieldPath: metadata.annotations['olm.targetNamespaces']
                 - name: ENABLE_CONVERSION_WEBHOOK
                   value: "true"
-                image: quay.io/argoprojlabs/argocd-operator:v0.12.0
+                image: quay.io/argoprojlabs/argocd-operator@sha256:f15781e2314d6baedcfc3bfa12181213d8d609803f73faecb2b8feddf3da4383
```

Comment:
Looks like image is built for arm, should be amd64
```bash
$ docker inspect quay.io/argoprojlabs/argocd-operator@sha256:f15781e2314d6baedcfc3bfa12181213d8d609803f73faecb2b8feddf3da4383 |grep Arch
          "Architecture": "arm64",
```

Reply:
Can we build multi-arch images ?

Reply:
Do we have a make command to build a multi-arch image? I did not configure any arch-type while building the image. I think it took my default laptop arch.

Reply:
Right now, no. You need to modify image build commands and add `--platform linux/amd64` flag.
```diff
diff --git a/Makefile b/Makefile
index 45c02a62..634356bb 100644
--- a/Makefile
+++ b/Makefile
@@ -110,7 +110,7 @@ run: manifests generate fmt vet ## Run a controller from your host.
        REDIS_CONFIG_PATH="build/redis" go run -ldflags=$(LD_FLAGS) ./cmd/main.go
 
 docker-build: test ## Build docker image with the manager.
-       $(CONTAINER_RUNTIME) build --build-arg LD_FLAGS=$(LD_FLAGS) -t ${IMG} .
+       $(CONTAINER_RUNTIME) build --platform linux/amd64 --build-arg LD_FLAGS=$(LD_FLAGS) -t ${IMG} .
 
 docker-push: ## Push docker image with the manager.
        $(CONTAINER_RUNTIME) push ${IMG}
```

Reply:
Thanks @svghadi !! Will build the image

Reply:
The image is updated on quay to be of type `amd64`.